### PR TITLE
Use <Pause> tag when saying token digits

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: two_factor/gateways/twilio/views.py [hunk 1]
```diff
@@ -23,13 +24,21 @@ class TwilioCallApp(View):
...
                        '  <Gather timeout="15" numDigits="1" finishOnKey="">'
-                       '    <Say language="%(locale)s">%(press_a_key)s</Say>'
+                       '    <Say language="{{ locale }}">{{ press_a_key }}</Say>'
                        '  </Gather>'
-                       '  <Say language="%(locale)s">%(no_input)s</Say>'
+                       '  <Say language="{{ locale }}">{{ no_input }}</Say>'
                        '</Response>',
         'token': '<?xml version="1.0" encoding="UTF-8" ?>'
                  '<Response>'
-                 '  <Say language="%(locale)s">%(token)s</Say>'
```
Comment (patroqueeet) [inline two_factor/gateways/twilio/views.py:32]: as we switch to Django templates anyway, I'd say moving it to file based templates might be more consequent. but besides that, I had reverted the same approach with Django templates :)

File: two_factor/gateways/twilio/views.py [hunk 2]
```diff
@@ -76,9 +95,4 @@ def get_twilio_locale(self):
     def get_prompt_context(self):
         return {
             'site_name': get_current_site(self.request).name,
-
```
Comment (patroqueeet) [inline two_factor/gateways/twilio/views.py:79]: I wonder if we now can drop this method at all. looks quite boilerplate like :)

**Review by patroqueeet — APPROVED** [review]
just smaller notes, but nothing serious from my perspective :)

**Comment by claudep** [comment]
@patroqueeet, I tried following your suggestions, seems good!

**Comment by patroqueeet** [comment]
> @patroqueeet, I tried following your suggestions, seems good!

I really like this code, esp. as the templates can now be customised through overriding :)

want me to check and contribute German parts?

**Comment by claudep** [comment]
> want me to check and contribute German parts?

You should be able to translate to German as soon as this patch is merged and the new strings are pushed to Transifex.

**Comment by patroqueeet** [comment]
> > want me to check and contribute German parts?
> 
> You should be able to translate to German as soon as this patch is merged and the new strings are pushed to Transifex.

can't wait for it :)

**Review by patroqueeet — APPROVED** [review]
LGTM! 🚢 !

**Comment by patroqueeet** [comment]
> @patroqueeet can you mark the conversations you started as resolved if they have been addressed by @claudep? I want to make sure there aren't any still outstanding requests.

@dopry GH doesn't offer me any buttons to approve/close. sry

**Comment by patroqueeet** [comment]
> @patroqueeet No worries! are all the open conversations resolved? I'll mark 'em closed. For ya.

I'd say so

**Review by moggers87 — APPROVED** [review]
LGTM, ignoring CodeCov as it's just being silly

**Comment by claudep** [comment]
@patroqueeet, new strings are available for translation on Transifex!

**Comment by patroqueeet** [comment]
@claudep I just send #502 for it. was that bad?

**Pull request merged**